[E 250310 10:51:24 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM2OTI5NDgiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pRc0luTnZkWEpqWlNJNklqTWlMQ0prWlhacFkyVmZhV1FpT2lJek0yUTBNek0yTVMxbFlUWmtMVE5sTTJVdFlUUmpZaTB5WXpkaU9UY3hOMlZqWVdJaUxDSnJhV1FpT2lKMGNtRmtaVjlyWlhsZmRqSWlMQ0p2Ylc1bGJXRnVZV2RsY21sa0lqbzBMQ0p3Y205a2RXTjBjeUk2ZXlKa1pXMWhkQ0k2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlN3aWJXWWlPbnNpYzNSaGRIVnpJam9pWVdOMGFYWmxJbjBzSW01aWRWOXNaVzVrYVc1bklqcDdJbk4wWVhSMWN5STZJbUZqZEdsMlpTSjlmU3dpYVhOeklqb2lkSEpoWkdWZmJHOW5hVzVmYzJWeWRtbGpaU0lzSW5OMVlpSTZJbE0yT1RJNU5EZ2lMQ0psZUhBaU9qRTNOREUyTmpnMU1qQXNJbTVpWmlJNk1UYzBNVFU0TVRrME1Dd2lhV0YwSWpveE56UXhOVGd4T1RRd0xDSnFkR2tpT2lJNE9HVXpZV1l6TnkwMk9URTNMVFJtTWpJdE9EVm1OUzAxTldSa09UZzFOV00yTWpJaUxDSlViMnRsYmlJNklpSjkudUkyb1JTNlpBVUxTTTloYkxTUGY2djM2UEJnWk1WTHR6V3JpNDZuc19nb2k0OTQ1TFlHeXZJQnFsbHhJRHhPWURXM1VIZkpBTHlvRVJjMk12ZnFNbnV4Y0R1NG1nTnFjak9EMWdzNU96bHNuRFJlMDV0RGZHd2pXMEtyWE5HT0ZKUkNrWXhxeXM2cEtNQWhucTRaOUt5dUVrVGZRakF5Nk1QdEtza2NvbTBvIiwiQVBJLUtFWSI6InVRQklnTUd2IiwiaWF0IjoxNzQxNTgyMTIxLCJleHAiOjE3NDE2Njg1MjB9.YnGVgkJp1IEvXRUuG2ieSzMEzFrooWy7HcOXnj7RfSAyNNa19Tw6mNUBJrVmQPcZfBvaKEWq3o0btEgJ2tZIWw'}, Request: {'exchange': 'BFO', 'tradingsymbol': 'SENSEX2531174700CE', 'symboltoken': '859312'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000020DFB66E0D0>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250310 11:50:00 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM2OTI5NDgiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pRc0luTnZkWEpqWlNJNklqTWlMQ0prWlhacFkyVmZhV1FpT2lJek0yUTBNek0yTVMxbFlUWmtMVE5sTTJVdFlUUmpZaTB5WXpkaU9UY3hOMlZqWVdJaUxDSnJhV1FpT2lKMGNtRmtaVjlyWlhsZmRqSWlMQ0p2Ylc1bGJXRnVZV2RsY21sa0lqbzBMQ0p3Y205a2RXTjBjeUk2ZXlKa1pXMWhkQ0k2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlN3aWJXWWlPbnNpYzNSaGRIVnpJam9pWVdOMGFYWmxJbjBzSW01aWRWOXNaVzVrYVc1bklqcDdJbk4wWVhSMWN5STZJbUZqZEdsMlpTSjlmU3dpYVhOeklqb2lkSEpoWkdWZmJHOW5hVzVmYzJWeWRtbGpaU0lzSW5OMVlpSTZJbE0yT1RJNU5EZ2lMQ0psZUhBaU9qRTNOREUyTnpBMk1EZ3NJbTVpWmlJNk1UYzBNVFU0TkRBeU9Dd2lhV0YwSWpveE56UXhOVGcwTURJNExDSnFkR2tpT2lKaU1tUTFOREptWmkwNFpXSTRMVFE1TW1RdFlqazBNUzFtWTJWa1kyTXdObUV6TlRRaUxDSlViMnRsYmlJNklpSjkucUNSVnEyaVlPYlEtY244WE54a0JEUjg4SzJOVURCX2lXZFZWUnRfV1p5SVZ5V3lMSzZzMUV4al9URm1JTGhTV1NyakNHeDhXX2dxdnhFenZXQWRCbjEzWWt1djZKeW1ZTl9YZ0tuWTVhSUZfSkFkb29oMkVuNGhOWkdpcGJFekhwV1ZKclhQUExiR3RPRjJOdTVIY1duSThyNUJKYlB1SUxMTFFFR2RoTEw0IiwiQVBJLUtFWSI6InVRQklnTUd2IiwiaWF0IjoxNzQxNTg0MjA4LCJleHAiOjE3NDE2NzA2MDh9.BK5STBN7FEXKWvk40d3-SeKm6A3wMLS75cvva6yaRuHkShigUoGC9WivzvRFkiqqIcZAT5jYqS4BCOFgAnllHg'}, Request: {'exchange': 'BSE', 'tradingsymbol': 'SENSEX', 'symboltoken': '99919000'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000011D0EE1E0D0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250310 13:10:52 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM2OTI5NDgiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pRc0luTnZkWEpqWlNJNklqTWlMQ0prWlhacFkyVmZhV1FpT2lJek0yUTBNek0yTVMxbFlUWmtMVE5sTTJVdFlUUmpZaTB5WXpkaU9UY3hOMlZqWVdJaUxDSnJhV1FpT2lKMGNtRmtaVjlyWlhsZmRqSWlMQ0p2Ylc1bGJXRnVZV2RsY21sa0lqbzBMQ0p3Y205a2RXTjBjeUk2ZXlKa1pXMWhkQ0k2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlN3aWJXWWlPbnNpYzNSaGRIVnpJam9pWVdOMGFYWmxJbjBzSW01aWRWOXNaVzVrYVc1bklqcDdJbk4wWVhSMWN5STZJbUZqZEdsMlpTSjlmU3dpYVhOeklqb2lkSEpoWkdWZmJHOW5hVzVmYzJWeWRtbGpaU0lzSW5OMVlpSTZJbE0yT1RJNU5EZ2lMQ0psZUhBaU9qRTNOREUyTnpZMU16QXNJbTVpWmlJNk1UYzBNVFU0T1RrMU1Dd2lhV0YwSWpveE56UXhOVGc1T1RVd0xDSnFkR2tpT2lJd016QmlObVF3TWkwMk56TXhMVFEzWVRBdE9ESmxOaTFtWmpJME1qWmtObU0zWkRRaUxDSlViMnRsYmlJNklpSjkuaW1HbjZNN2k5d0JMLTBYbFl1c1J3YTlsZ2hZejlDbW1XeDYxZWdtbjRNMlk3YnQ1eGNLbGFJQUlvVnQ3Wjl3Vzc1aVhTUlFqSTRyekFrSUltSlBlZEtiS194ZmZNSDBTNkluOFltR1ZpVkpjX1N2LWwxdWVZTTcxQ1djTWE2UTRVQjJNRkZ3Z195OThFNk5MZ2xLVThXUnVudzNQN2xWc2I5VllWUVQxUkZVIiwiQVBJLUtFWSI6InVRQklnTUd2IiwiaWF0IjoxNzQxNTkwMTMxLCJleHAiOjE3NDE2NzY1MzB9.oR8PU9rd02ZzvdQCQlZttfRWJ2D4sAROKIAOZIZuAaXihoMP9J_ttIJCF_fyx4fd90YoeMnHLDVYuZG1kj3oZA'}, Request: {'exchange': 'BFO', 'tradingsymbol': 'SENSEX2531174600CE', 'symboltoken': '858905'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001E7AA3DE0D0>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
